server/task_executor_pitr_cutover.go

```diff
@@ -98,7 +98,7 @@ func (exec *PITRCutoverTaskExecutor) pitrCutover(ctx context.Context, task *api.
 	}
 	defer driver.Close(ctx)
 
-	if err := exec.doCutover(ctx, driver, server, task, issue); err != nil {
+	if err := exec.doCutover(ctx, driver, task, issue); err != nil {
```

Comment:
I think it's more readable to do it regardless of the database types.

Reply:
SGTM.